scala/scalafmt/BUILD

```diff
@@ -0,0 +1,37 @@
+load("//scala:scala.bzl", "scala_binary")
+
+filegroup(
+    name = "runner",
+    srcs = ["private/format.template.sh"],
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "testrunner",
+    srcs = ["private/format-test.template.sh"],
+    visibility = ["//visibility:public"],
+)
+
+scala_binary(
+    name = "scalafmt",
+    srcs = glob(["scalafmt/ScalafmtRunner.scala"]),
```

Comment:
plain srcs? you're not globbing anything here

Reply:
Good catch, thanks.